### DISCUSSION TITLE
Announcing the new Vercel Community

### DISCUSSION THREAD

**Discussion by pawlean** [vercel/community#7413] [Announcements]
Hi all,

On July 15th, 2024, we’ll be starting the process of **migrating from GitHub Discussions** to our **own Vercel Community**.

We’re doing this to provide a much improved community experience and focus, bringing all of you closer to Vercel. You’ll be able to access more information, education, and opportunities to collaborate with the Vercel team so everyone can succeed using Vercel.

The migration will be completed on the following timeline:

- **7/15 -** We’ll open the new Vercel Community for registrations (this will be with your Vercel account) and you will no longer be able to create new GitHub Discussions. Stay tuned for access to the new community platform!
- **9/16 -** Existing GitHub Discussions (`vercel/community` and `vercel/vercel`) will be made *read-only*.
- **12/16 -** GitHub Discussions (`vercel/community` and `vercel/vercel`) will be turned off and no longer publicly available.

We’re confident this change will lead an improved experience for all involved and have significant plans for 2025 to provide even greater support to our community.

Please let us know if you have any questions, thoughts, or feedback!

**EDIT:** We're live at [vercel.community](http://vercel.community/). See you there! 🥳

**Comment by amyegan** [comment]
I am beyond excited to finally be able to share this project with you all! 🥳

**Comment by Arindam200** [comment]
This Sounds Pretty Cool. Excited for this!

**Comment by Ansub** [comment]
Godspeed!

**Comment by TGlide** [comment]
Will existing discussions be migrated to the new platform? It'd be a shame to eventually lose the current discussions.

**Reply by pawlean** [reply]
We're not planning to migrate existing discussions to the new platform unfortunately, but you still have access to it until the end of the year. We'll also do our best until then to get as much of the useful information here over there as possible.

**Comment by pawlean** [comment]
And we’re live! Let’s continue our discussions at **[vercel.community](http://vercel.community/)**. See you there! 🥳

**Reply by stefanofa** [reply]
Hey @pawlean , I tried to sign in but I always get "Something went wrong."
![image](https://github.com/user-attachments/assets/3b96e451-65e9-49cd-a713-2f70c5b4f094)
![image](https://github.com/user-attachments/assets/0ca1cdc7-cab3-4585-ba67-9447fef6ef56)

**Reply by pawlean** [reply]
Hi @stefanofa!

Thanks for sharing. We're looking into it.

**Reply by javierbyte** [reply]
@stefanofa could you please try again? 😄

**Reply by stefanofa** [reply]
@javierbyte worked now! thanks for the quick fix 🤩

**Comment by NolanLan** [comment]
I need help.this is my Vercel account:GY6vKL5Lkhjb187bg3XZkiwg
when i deploy,hava an error:Your Team encountered an unknown problem. Please reach out to our support team for details

**Reply by pawlean** [reply]
Hi @NolanLan! 

You should be good to go. Moving forward, would love to see you on our [new platform.](https://vercel.community)!

**Comment by aryan2621** [comment]
Vercel Id: UA6gPiSHV9bfx32g4lR6HiDW

Your Team encountered an unknown problem. Please reach out to our support team for details.

**Reply by pawlean** [reply]
Hi @aryan2621!

That should work for you now. Let's continue the conversation over on our [new platform](https://vercel.community)!
